tests/covalent_dispatcher_tests/_core/execution_test.py

```diff
@@ -37,6 +39,28 @@
 
 TEST_RESULTS_DIR = "/tmp/results"
 
+DEFAULT_THREADS_PER_WORKERS = 1
+DEFAULT_N_WORKERS = dask.system.CPU_COUNT
+DEFAULT_MEM_PER_WORKER = "auto"
+
+
+@pytest.fixture
+def test_cluster():
+    cluster = LocalCluster(
+        n_workers=DEFAULT_N_WORKERS,
+        threads_per_worker=DEFAULT_THREADS_PER_WORKERS,
+        **{"memory_limit": DEFAULT_MEM_PER_WORKER},
+    )
+    yield cluster
+    cluster.close()
+
+
+@pytest.fixture
+def event_loop():
+    loop = asyncio.new_event_loop()
+    yield loop
+    loop.close()
```

Comment:
Are you sure we need this fixture? The event loop should get automatically passed to the asyncio marked test as mentioned here: https://pytest-asyncio.readthedocs.io/en/latest/concepts.html.

Reply:
@kessler-frost I remember this test being particularly brittle because pytest-asyncio likes to close the event loop prematurely.

Reply:
This was probably because of using a `DaskExecutor`.
I have switched to `LocalExecutor` though.